lunabot_embedded/src/teensy_driver_node.cpp

```diff
@@ -92,8 +90,7 @@ void recv(ros::Publisher &pub) {
   state_msg.drive_right_ang = DEG2RAD(state.drive_right_ang);
   state_msg.drive_left_vel = DEG2RAD(left_vel);
   state_msg.drive_right_vel = DEG2RAD(right_vel);
-  state_msg.dep_ang = DEG2RAD(state.dep_ang);
-  state_msg.lead_screw_ang = DEG2RAD(state.lead_screw_ang);
+  state_msg.exc_ang = DEG2RAD(state.exc_ang);
   state_msg.uwb_dists.push_back(state.uwb_dist_0);
   state_msg.uwb_dists.push_back(state.uwb_dist_1);
   state_msg.uwb_dists.push_back(state.uwb_dist_2);
```

Comment:
load_cell_weights aren't updated here

Reply:
That's correct, there is no load cell code in this entire pull request